### PR TITLE
Add support for `--clients` to `pbench-linpack`

### PR DESCRIPTION
**Pull request by portante**
Replaces PR #2729, tracked in Jira as PBENCH-170.

### CODE REVIEW

File: agent/bench-scripts/pbench-linpack [hunk 1]
```diff
@@ -201,25 +209,56 @@ for thread in ${threads//,/ }; do
 	mkdir -p "${iteration_dir}"
 	record_iteration ${count} ${thread} ${iteration}
 	echo "Starting iteration ${iteration}"
-
-	# Pre-processing: generate the linpack.input dirs (in /var/lib/pbench-agent/tmp)
+	# Pre-processing: generate the linpack.input files
 	preprocess-iteration "${iteration_dir}" "${thread}"
+	# Distribute the linpack.input files to remote clients if necessary
```
Comment (webbnh) [inline agent/bench-scripts/pbench-linpack:214]: If you need to make a change for some other reason, I suggest replacing `linpack.input` with `linpack input` (replace the dot with a space).

**Review by webbnh — DISMISSED** [review]
One nit, which you can take or leave.

File: agent/bench-scripts/pbench-linpack (hunk 1 above)
Comment (portante) [reply agent/bench-scripts/pbench-linpack]: Fixed.

**Review by portante — COMMENTED** [review]
Okay, we're ready!

File: agent/bench-scripts/pbench-linpack (hunk 1 above)
Comment (webbnh) [reply agent/bench-scripts/pbench-linpack]: You got one of them...you didn't want to fix the other?  🤪

**Review by webbnh — APPROVED** [review]
Oh, so close.

**Review by portante — COMMENTED** [review]
Okay, ready now!

File: agent/bench-scripts/pbench-linpack (hunk 1 above)
Comment (portante) [reply agent/bench-scripts/pbench-linpack]: Fixed.

**Review by webbnh — APPROVED** [review]
👍

**Review by npalaska — APPROVED** [review]
looks as good as #2729 plus all the tests :+1:

**Pull request merged**